File: src/main/java/RemoveElement/L07152055/RemoveElement.java

class Solution {
    public int removeElement(int[] nums, int val) {
        int length = nums.length;
        int j = 0;
        for(int i = 0; i < nums.length; i++) {
            if(nums[i] != val) {
                nums[j++] = nums[i];
            } else {
                length--;
            }
        }
        return length;
    }
}

File: src/main/java/RemoveElement/Ardax/RemoveElement.py
class Solution(object):
    def removeElement(self, nums, val):
        """
        :type nums: List[int]
        :type val: int
        :rtype: int
        """
        index=0
        for i in nums:
            if i!=val:
                nums[index]=i
                index+=1
        return index